application/__init__.py

```diff
@@ -17,13 +17,21 @@
 #   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
 #   FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
 #   IN THE SOFTWARE.
-
+import logging.config
+import yaml
 import os
 import sys
 
 from flask import Flask, Blueprint
 from werkzeug.serving import WSGIRequestHandler
 
+with open('logging_default_config.yaml', 'rt') as config_file:
```

Comment:
Let's wrap the `open()` and `logging.config.dictConfig()` code with a `try-catch` in the case the config file does not exist or is corrupted. We can `print("Logging setup failed with exception = ", ex)`.